examples/live/check_results.py

```diff
@@ -1,208 +1,236 @@
-            sngl_inspiral_table = lsctables.SnglInspiralTable.get_table(xmldoc)    
-            end_time=sngl_inspiral_table.get_column('end_time')  
-            snr_list=sngl_inspiral_table.get_column('snr')
-            network_snr_squared=0
-            for snr in snr_list:
-                  network_snr_squared+=snr**2
-            network_snr=np.sqrt(network_snr_squared)            
-            mass1=sngl_inspiral_table.get_column('mass1')[0]
-            mass2=sngl_inspiral_table.get_column('mass2')[0]    
-            spin1z=sngl_inspiral_table.get_column('spin1z')[0] 
-            spin2z=sngl_inspiral_table.get_column('spin2z')[0] 
-            log.info('IFO SNRs: '+str(snr_list))
-            log.info('Network SNR: '+str(network_snr))
-            
-            if print_template_parameters:
-                  log.info('IFO End Times: '+str(end_time)) 
-                  log.info('Mass 1: %f', mass1)
-                  log.info('Mass 2: %f', mass2)
-                  log.info('Spin1z: %f', spin1z)
-                  log.info('Spin2z: %f', spin2z)
-            
-            #check if trigger properties match injection's properties
-            for t in end_time:
-                  if not close(t,inj_time,1.0):    
-                        coinc_fail = True
-                        log.error('Trigger time does not match injection time')  
-            if not close(mass1, inj_mass1, 5e-7):
-                  coinc_fail = True
-                  log.error('Trigger mass 1 does not match injection mass 1')
-            if not close(mass2, inj_mass2, 5e-7):
-                  coinc_fail = True                  
-                  log.error('Trigger mass 2 does not match injection mass 2')
-            if not close(spin1z, inj_spin1z, 5e-7):
-                  coinc_fail = True
-                  log.error('Trigger spin1z does not match injection spin1z')
-            if not close(spin2z, inj_spin2z, 5e-7):
-                  coinc_fail = True
-                  log.error('Trigger spin2z does not match injection spin1z')   
-            if not close(network_snr, inj_snr, 0.1):
-                  coinc_fail = True
-                  log.error('Network SNR test failed')
+        sngl_inspiral_table = lsctables.SnglInspiralTable.get_table(xmldoc)
+        
+        trig_props['tc'][x]=sngl_inspiral_table.get_column('end_time')[0]                   
+        trig_props['mass1'][x]=sngl_inspiral_table.get_column('mass1')[0]
+        trig_props['mass2'][x]=sngl_inspiral_table.get_column('mass2')[0]    
+        trig_props['spin1z'][x]=sngl_inspiral_table.get_column('spin1z')[0] 
+        trig_props['spin2z'][x]=sngl_inspiral_table.get_column('spin2z')[0] 
+        
+        
+        snr_list=sngl_inspiral_table.get_column('snr')
+        network_snr_squared=0
+        for snr in snr_list:
+            network_snr_squared+=snr**2
+        network_snr=np.sqrt(network_snr_squared)         
```

Comment:
Another Python trick to shorten your code (and make it faster, though it will not be an issue here) is list comprehension:
```python
network_snr = sum([snr ** 2 for snr in snr_list]) ** 0.5
```

Reply:
and if `snr_list` is actually a Numpy array (I forget what `get_column()` returns) you can even do
```python
network_snr = sum(snr_list ** 2) ** 0.5
```